cdk/domino_cdk/config/template.py

```diff
@@ -78,6 +78,8 @@ def add_nodegroups(name, count, min_size, instance_types, labels, disk_size=100,
         id=None,
         create=True,
         cidr='10.0.0.0/16',
+        public_cidr_mask=27,
```

Comment:
In 208.128.0.0/**11** and **255.224.0.0** "11" is called prefix and "255.224.0.0" is called mask.

Reply:
I believe `208.120.0.0` is the prefix, `255.224.0.0` us the subnet mask, and `11` is just the cidr notation version of 255.224.0.0 (ie you could read it as `208.128.0.0/255.224.0.0`--they mean the same thing).

Anyhow, the choice is naming is to reflect the api call to `SubnetConfiguration`, which uses `cidr_mask` to name its value (see vpc provisioner in this same PR). I don't believe the terminology here is inaccurate (subnet mask given as cidr value), and would prefer to keep the name mapping the same as the API.